src/ingest-pipeline/airflow/dags/salmon_rnaseq.py

```diff
@@ -167,12 +208,23 @@ def build_cwltool_cmd2(**kwargs):
             python_callable=utils.pythonop_maybe_keep,
             provide_context=True,
             op_kwargs={
-                "next_op": "move_data",
+                "next_op": "prepare_cwl3",
                 "bail_op": "set_dataset_error",
                 "test_op": "t_convert_for_ui",
             },
         )
 
+        t_maybe_keep_cwl3 = BranchPythonOperator(
+            task_id="maybe_keep_cwl3",
+            python_callable=utils.pythonop_maybe_keep,
+            provide_context=True,
+            op_kwargs={
+                "next_op": "move_data",
+                "bail_op": "set_dataset_error",
+                "test_op": "t_convert_for_ui_2",
```

Comment:
test_op should be "convert_for_ui_2", the task_id of the step.  t_convert_for_ui_2 is the name of the python variable pointing to the step.